lib/ddtrace/configuration/settings.rb

```diff
@@ -32,7 +32,13 @@ class Settings
       end
 
       settings :diagnostics do
-        option :debug, default: false
+        option :debug do |o|
+          o.default false
+          o.on_set do |enabled|
+            # Enable rich debug print statements
+            require 'pp' if enabled
```

Comment:
Loading of `pp` is now lazy, as it is only used in production `ddtrace` when debug diagnostics are enabled. 

Although `pp` is commonly used in testing and development, it might not be loaded by the host application in production.